napari/utils/migrations.py

```diff
@@ -4,7 +4,9 @@
 from napari.utils.translations import trans
 
 
-def rename_argument(from_name: str, to_name: str, version: str):
+def rename_argument(
+    from_name: str, to_name: str, version: str, since_version: str = "unknown"
+):
```

Comment:
Want to make `since_version` mandatory ?

Reply:
In general yes, but it becomes a part of public API. But maybe I could assume that no one outside napari is using it.

Reply:
Fair. Want to emit a FutureWarning it will be mandatory ?

Reply:
Yes. I will add this.